tests/unit/s2n_handshake_test.c

```diff
@@ -322,12 +328,10 @@ int main(int argc, char **argv)
 
         EXPECT_NOT_NULL(server_config = s2n_config_new());
         EXPECT_SUCCESS(s2n_config_set_cipher_preferences(server_config, "test_all_tls13"));
-        EXPECT_SUCCESS(s2n_config_set_signature_preferences(server_config, "20200207"));
```

Comment:
The specific signature preferences are important here. The list MUST contain RSA_PSS. Can we guarantee that the default does?

Reply:
"test_all_tls13" version corresponds to `security_policy_test_all_tls13` security policy which contains `s2n_signature_preferences_20200207` as the sign_pref. This sign_pref includes RSA_PSS. If we want to be more specific, I can add a version set called `20200207` to the list.